tests/test_fsclient.py

```diff
@@ -42,15 +38,108 @@
 DEFAULT_RC = {}
 rc = DEFAULT_RC
 
+
 # FileSystemClient methods tested here
 def test_is_alive():
-    expected = True  #filesystem is always alive!
+    expected = True  # filesystem is always alive!
     fsc = FileSystemClient(rc)
     actual = fsc.is_alive()
+
     assert actual == expected
 
-@pytest.mark.skip("Not written")
+
 def test_open():
+    fsc = FileSystemClient(rc)
+    fsc.open()
+
+    assert isinstance(fsc.dbs, type(defaultdict(lambda: defaultdict(dict))))
+    assert isinstance(fsc.chained_db, type(dict()))
+    assert not fsc.closed
+
+
+@pytest.mark.skip("Not written")
+def test_load_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_cif():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_load_database():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_json():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_yaml():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_cif():
+    pass
+
+
+@pytest.mark.skip("Not written")
+def test_dump_database():
+    pass
+
+
+def test_close():
+    fsc = FileSystemClient(rc)
+    fsc.close()
+
```

Comment:
I think close this up?  Not sure what pep8 has to say about this, but it keeps the tests grouped visually  bit better.  Do what pep8 says but close up if it has nothign to say.